de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/editor/EditorManager.java

```diff
@@ -399,6 +400,7 @@ private void startSession(ISarosSession newSarosSession) {
           session.addActivityConsumer(consumer, Priority.ACTIVE);
 
           documentListener.setEnabled(true);
+          annotationDocumentListener.setEnabled(true);
```

Comment:
Maybe for later. If I observe this correctly is that what you really need is a disable-"bare" broadcaster, e.g

editorManager().getDocumentListenerBroadCast().addListener(…)

As there are currently only 2 listeners I do not see a reason for doing this now.